frontend/packages/console-app/src/components/console-operator/ConsoleOperatorConfig.tsx

```diff
@@ -80,6 +83,43 @@ const ConsolePluginStatus: React.FC<ConsolePluginStatusType> = ({ enabled, plugi
 
 const ConsolePluginsList: React.FC<ConsolePluginsListType> = ({ obj }) => {
   const { t } = useTranslation();
+  const toastContext = useToast();
+  const [pluginsData, pluginsError] = useURLPoll<ConsolePluginsListResponseType>(
```

Comment:
We'd want the toast notification to show up on any page of console, so we'd need to move the logic elsewhere (maybe to the masthead?)

Reply:
@spadgett good point... FYI `masthead.jsx` and `masthead-toolbar.jsx` are both JSX and not TSX. I'm assuming we don't want to convert those and drop the typescript functionality for now?

Reply:
Thinking more on this, the top-level `App` component might be better, but that's also JSX.

masthead-toolbar is large enough that we probably want a separate tech debt story to update it.

Reply:
@spadgett done. I have moved the logic to the `app.jsx` container.